src/main/java/io/reactivex/exceptions/OnCompleteFailedException.java

```diff
@@ -18,7 +18,11 @@
     private static final long serialVersionUID = -6179993283427447098L;
 
     public OnCompleteFailedException(Throwable cause) {
-        super(cause);
+        super(cause != null ? cause : new NullPointerException());
```

Comment:
Here, it is an indication that somewhere the cause wasn't set properly, the NPE will help identify that.